index.bs

```diff
@@ -373,6 +373,22 @@ The <dfn attribute for="Document">onprerenderingchange</dfn> attribute is an [=e
 
 As is customary for [[HTML]], the definition of {{Document/prerendering}} would be located in another section of the spec; we'd place it in the new section introduced below:
 
+<h3 id="performance-navigation-timing-extension">Extensions to the {{PerformanceNavigationTiming}} interface</h3>
+
+We'd modify the definition of {{PerformanceNavigationTiming}} as follows:
```

Comment:
I think we can say "Extend the {{PerformanceNavigationTiming}} interface as follows:"